src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -667,8 +692,10 @@ public synchronized void initServer(int delay) throws ConfigurationException
         {
             logger.info("Loading persisted ring state");
             populatePeerTokenMetadata();
+            Multimap<InetAddressAndPort, Token> loadedTokens = SystemKeyspace.loadTokens();
```

Comment:
this happened after rebasing to pick up CASSANDRA-16246, thinking I should avoid loading a second time and rely on TokenMetadata for this information (CASSANDRA-16246 makes sure token + host_id are present)